### PR TITLE
Made add product functionality

### PR DESCRIPTION
**Pull request by carterclark**

### CODE REVIEW

**Review by paveldanek — COMMENTED** [review]
Hey, I'm not sure why you changed validateProductId() to validateProduct(). You also don't need to parse getString to int, there's a method getInt() there. So is getDouble(). Also, after adding a product, it has to be ordered. There's a method in GroceryStore for it. It's called reorderProduct(). Good job with making sure that the product name doesn't exist before adding. By the way, wasn't this supposed to be Ricky's functionality?

**Review by paveldanek — APPROVED** [review]
That looks pretty good.

File: src/store/facade/GroceryStore.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public boolean remove(String id) {
 		 * @param id - the ID of the member searched for
 		 * @return Member object if found, null if not found
 		 */
-		public Member search(String id) {
```
Comment (carterclark) [inline src/store/facade/GroceryStore.java:85]: this needs to be different

File: src/store/facade/GroceryStore.java [hunk 2]
```diff
@@ -136,7 +137,7 @@ public String add(Product product) {
 		 * @param id - the ID of the product searched for
 		 * @return Product object if found, null if not found
 		 */
-		public Product search(String id) {
+		public Product searchById(String id) {
```
Comment (carterclark) [inline src/store/facade/GroceryStore.java:140]: review

File: src/store/facade/GroceryStore.java [hunk 3]
```diff
@@ -146,6 +147,16 @@ public Product search(String id) {
 			return null;
 		}
```
Comment (carterclark) [inline src/store/facade/GroceryStore.java:149]: yes

**Pull request merged**